src/components/Header/Header.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+
+import { BottomNavigation, BottomNavigationAction, Box } from '@mui/material'
+import React from 'react'
+
+interface Props {}
+
+const style = {
+  link: {
+    fontSize: 14,
+    fontWeight: '600',
+    cursor: 'pointer',
+    borderRadius: '6px',
+    paddingX: '0.8rem',
+    paddingY: '0.4rem',
+    '&:hover': {
+      transform: 'translateY(0.4rem)',
+      backgroundColor: 'black',
+      color: 'white',
+    },
+    '&:active': {
+      transform: 'scale(0.90) translateY(0.4rem)',
+    },
+    '& .Mui-selected': {
+      color: '#2563eb',
+    },
+    transition: 'all 240ms cubic-bezier(0.4, 0.4, 0.4, 0.4)',
+  },
+}
+
+function Header({}: Props) {
+  const [value, setValue] = React.useState(0)
+  return (
+    <Box
+      component={'header'}
+      sx={{
+        width: '100vw',
+        display: 'flex',
+        justifyContent: 'center',
+        borderBottom: 2,
+        paddingY: '1.2rem',
+        borderColor: '#e7e5e4',
+      }}
+    >
+      <Box sx={{ display: 'flex', flexDirection: 'column', rowGap: '0.4rem' }}>
+        <Box
+          sx={{
+            fontSize: 28,
+            fontWeight: '800',
+          }}
+        >
+          OpenBC
+        </Box>
```

Comment:
Not that I think it really matters all that much, but just noting that there's a slight difference between the implementation and the design as far as how these line up:

### Current Commit

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/ad300cb7-2959-4f53-99e7-0d2c3f743fea)

### Design

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/ae6581fa-4faa-47e9-999f-8c26dd372b1c)

Reply:
The design haven't thought about the button hovers and animations. But I do.

Reply:
Maybe we can look at this later with UX/UI in mind.

Reply:
I agree with Correy on this, it looks off if ppl don't click on it. Maybe center 'OpenBC' for now?

Reply:
I agree with Nam. Center looks so much better now.
![image](https://github.com/OpenBCca/openbc-web/assets/72838071/f2fc432a-4836-410b-9f07-ce82ffef99c4)